exonum-java-binding/common/src/test/java/com/exonum/binding/common/proofs/DbKeyCompressedFunnelTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.proofs;
+
+import static com.exonum.binding.common.proofs.DbKeyCompressedFunnel.getWholeBytesKeyLength;
+import static com.exonum.binding.common.proofs.map.DbKeyTestUtils.keyFromString;
+import static org.mockito.Mockito.anyByte;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.exonum.binding.common.hash.PrimitiveSink;
+import com.exonum.binding.common.proofs.map.DbKey;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class DbKeyCompressedFunnelTest {
+
+  @ParameterizedTest
+  @MethodSource("testSource")
+  void funnelTest(DbKey dbKey, byte[] encodedSignificantBitsNum) {
+    PrimitiveSink primitiveSink = mock(PrimitiveSink.class);
+    DbKeyCompressedFunnel.dbKeyCompressedFunnel().funnel(dbKey, primitiveSink);
+    int wholeBytesKeyLength = getWholeBytesKeyLength(dbKey.getNumSignificantBits());
```

Comment:
Why getWholeBytesKeyLength is still used in the test to get the **expected** value or not tested separately?